install-manifests/docker-compose-traefix-tls-gzip/README.md

```diff
@@ -0,0 +1,29 @@
+# Hasura GraphQL Engine on Docker with Traefix (Reverse Proxy and Load Balancer)
```

Comment:
Traefix should be `Traefik` as it is the official name

Reply:
Done.